src/Cake.Core.Tests/Fixtures/NotFormattableFixture.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Cake.Core.Tests.Fixtures
+{
+    internal sealed class NotFormattableFixture
+    {
+        public static NotFormattableFixture Default { get; } = new NotFormattableFixture();
```

Comment:
**suggestion (naming):** I tend to avoid using the word "_default_" when naming things, since it doesn't really set any expectations.

If the role of this property is to expose a singleton, I would suggest calling it something more specific like `Instance`.

Reply:
this should now be resolved